recipes/chrpath/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "0.16" %}
+
+package:
+  name: chrpath
+  version: {{ version }}
+
+source:
+  fn: chrpath-{{ version }}.tar.gz
+  url: http://pkgs.fedoraproject.org/repo/pkgs/chrpath/chrpath-{{ version }}.tar.gz/2bf8d1d1ee345fc8a7915576f5649982/chrpath-{{ version }}.tar.gz
```

Comment:
Guess this is an `md5` checksum. We could probably template that out as well for ease.